src/store/actions/notifier.js

```diff
@@ -330,14 +330,14 @@ const manageNewChannel = async (notification, notifier) => {
   );
   const selfAddress = getState().client.address;
 
-  // NOTE: The opener of the channel is the value[1] which is the first address
+  // NOTE: The opener of the channel is the value[2] which is the first address
 
-  let partner_address = getAddress(values[1].value);
+  let partner_address = getAddress(values[2].value);
```

Comment:
This makes sense but we should return named parameters from the notifier, its confusing @rorolopetegui